config/rbac/dedicated-admin-role.yaml

```diff
@@ -0,0 +1,19 @@
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: df-dedicated-admin-role
+  labels:
+    managed.openshift.io/aggregate-to-dedicated-admins: cluster
```

Comment:
@v-harihar we are using aggregationRule here, if we add a role containing the label mentioned in aggregationRule it will add to that existing user role's rule
https://github.com/openshift/managed-cluster-config/blob/f4537242f9c54ee81bd049c458f3dc1e7ed9f838/deploy/rbac-permissions-operator-config/03-dedicated-admins-cluster.ClusterRole.yaml

https://kubernetes.io/docs/reference/access-authn-authz/rbac/#aggregated-clusterroles

Reply:
We can add this as a comment as well. Will be helpful IMO.

Reply:
sure @SanjalKatiyar